### PR TITLE
feat(viewer): collapse secondary session sections

### LINKED ISSUE #734: Suggestion: viewer should collapse all sections except `Review Comments`
### Problem Statement

As a code reviewer, the most important part is `Review Comments`, others are not that important, it's better collapse others by default (like the [Accordion component of bootstrap theme](https://getbootstrap.com/docs/5.3/components/accordion/)):

- FILE
- Files Reviewed
- Conversations (x files)

### Proposed Solution

collapse 3 sections.

### Alternatives Considered

_No response_

### Affected Area

WebUI Viewer

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Linxiushen**
## Description

Keep review findings prominent by collapsing the three secondary session details by default:

- per-file token usage
- files reviewed
- raw LLM conversations

The Viewer uses native `details` elements and the existing disclosure styling, so each section remains keyboard-accessible and can be expanded without JavaScript. Review Comments and their file groups remain open. Count badges keep the hidden scope visible at a glance, and the token table is placed in an overflow-safe body for narrow screens.

## Type of Change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring (no functional changes)
- [ ] Documentation update
- [ ] CI / Build / Tooling

## How Has This Been Tested?

- [x] Focused Viewer template/static tests
- [x] `go vet ./...`
- [x] Manual browser testing at desktop and mobile widths

The regression test verifies all three secondary sections render closed while Review Comment file groups remain open. Browser checks confirmed the sections stay closed at 1265px and 390px widths and expand normally when activated.

`go test ./internal/viewer` also reaches three existing permission-mode assertions that are not portable to Windows (`TestHandleRepos_PermissionDenied`, `TestDiscoverRepos_SkipsUnreadableSubdir`, and `TestListSessions_SkipsUnreadableFiles`). The focused Viewer rendering tests pass.

## Checklist

- [x] My code follows the project's coding style (`go fmt`, `go vet`)
- [x] I have performed a self-review of my code
- [x] I have added tests that prove the new default state
- [x] Relevant tests pass locally with my changes
- [x] Documentation is unchanged because the controls use native disclosure behavior
- [ ] I have signed the CLA

## Related Issues

Closes #734

### CODE REVIEW

File: internal/viewer/static/style.css [hunk 1]
```diff
@@ -420,6 +420,26 @@ h3 {
...
+    cursor: pointer;
+    font-size: 0.82rem;
+    font-weight: 600;
+    list-style: none;
+    padding: 0.25rem 0 0.75rem;
+}
+.token-breakdown-toggle::-webkit-details-marker { display: none; }
+.token-breakdown-toggle::marker { content: none; }
+.token-breakdown-toggle .file-count-badge { margin-left: auto; }
+.token-breakdown[open] > .token-breakdown-toggle .chevron-sm { transform: rotate(45deg); }
```
[the remark this answers is not in the document]
Comment (Linxiushen) [reply internal/viewer/static/style.css:437]: The class is already defined later in this stylesheet at .chevron-sm (around line 797). The token breakdown reuses that definition, and this selector only controls its open-state rotation, so no additional rule is needed here.

File: internal/viewer/templates/session.html [hunk 2]
```diff
@@ -80,20 +80,29 @@ <h3>Token Usage</h3>
...
-                <td title="{{.CompletionTokens}}">{{formatNumber .CompletionTokens}}</td>
-                {{if or $.Session.TokenUsage.TotalCacheReadTokens $.Session.TokenUsage.TotalCacheWriteTokens}}<td title="{{.CacheReadTokens}}">{{formatNumber .CacheReadTokens}}</td><td title="{{.CacheWriteTokens}}">{{formatNumber .CacheWriteTokens}}</td>{{end}}
-                <td title="{{add .PromptTokens .CompletionTokens}}"><strong>{{formatNumber (add .PromptTokens .CompletionTokens)}}</strong></td>
-            </tr>
-        {{end}}
-        </tbody>
-    </table>
+    <details class="token-breakdown">
+        <summary class="token-breakdown-toggle">
+            <span class="chevron-sm"></span>
```
[the remark this answers is not in the document]
Comment (Linxiushen) [reply internal/viewer/templates/session.html:85]: The smaller chevron is intentional here. Token breakdown is a compact nested disclosure inside the summary panel, while the 20x20 badge chevron marks top-level page sections; using the full section control here would give the nested row too much visual weight.

File: internal/viewer/templates/session.html [hunk 3]
```diff
@@ -151,16 +160,30 @@ <h3>Review Comments ({{len .Session.Comments}} findings)</h3>
...
 {{end}}
 
-<h3>Conversations ({{len .Session.Files}} files)</h3>
-<div class="conversations">
+<details class="file-accordion section-accordion">
+    <summary class="file-accordion-header">
+        <span class="chevron"></span>
+        <span class="section-title">Conversations</span>
+        <span class="file-count-badge">{{len .Session.Files}} files</span>
+    </summary>
```
[the remark this answers is not in the document]
Comment (Linxiushen) [reply internal/viewer/templates/session.html:185]: This reuse is intentional: the outer section should inherit the established accordion border, background, and interaction treatment, while .section-accordion provides the section-level spacing and nested-content reset. A separate class would duplicate the same base styles without changing the rendered behavior.

File: internal/viewer/templates/session.html [hunk 4]
```diff
@@ -151,16 +160,30 @@ <h3>Review Comments ({{len .Session.Comments}} findings)</h3>
...
+    <div class="file-accordion-body">
+        <div class="conversations">
```
[the remark this answers is not in the document]
Comment (Linxiushen) [reply internal/viewer/templates/session.html:187]: Fixed in 0e0cb81 by guarding the Conversations accordion with {{if .Session.Files}}. I also added a regression test that renders an empty session and verifies the section is omitted.

[the remark this answers is not in the document]
Comment (Linxiushen) [reply internal/viewer/templates/session.html:187]: Fixed in 0e0cb81 by guarding the Conversations accordion with {{if .Session.Files}}. The accompanying regression test verifies that an empty session does not render the section title.

[the remark this answers is not in the document]
Comment (Linxiushen) [reply internal/viewer/templates/session.html:187]: The shared base class is deliberate so both levels use the same established accordion treatment. The outer .section-accordion adds section spacing, and .section-accordion .conversations { margin: 0; } prevents compounded content spacing; the nested borders communicate the section/file hierarchy rather than being accidental leakage.

**Review by lizhengfeng101 — APPROVED** [review]
LGTM
